README.md

```diff
@@ -10,7 +10,7 @@ You need to have Docker installed on your machine. You can find instructions on
 
 ## Installation
 
-
+First, download the [latest release](https://github.com/rails/rails-new/releases/tag/latest) for your operating system. Click on the downloaded `.tar.gz` file to unzip it, which will create the `rails-new` executable. Move the executable into your path so that it is ready to run from the command line.
```

Comment:
We can also consider adding the git clone command

Reply:
Oops yeah I meant *does not* work, I'll edit it. 

You can use this link for now: https://github.com/rails/rails-new/releases/tag/v0.3.0